packages/web-components/@sfx/sayt/src/sayt.ts

```diff
@@ -187,6 +197,20 @@ export default class Sayt extends LitElement {
     this.requestSaytProducts(query, searchbox);
   }
 
+  /**
+   * Debounces the function that is passed in the callback paramater.
+   * Customizable delay based on debounceTime attribute on Sayt element.
+   *
+   * @param callback The function to debounce.
+   */
+  debounceSaytRequests(callback: (query: string, searchbox?: string) => void) {
```

Comment:
My feeling here is that `debounceSaytRequests` doesn't care about the type of the callback.

Either way, we've been leaving out void return type, and just letting that be implicit if there is no provided return type.

I'm wondering if this method should just be called `getDebounce()` or something. Although we use it for debouncing SAYT requests, it doesn't really matter what uses it. I suppose we do use `this.debounceTime`, but even that is not specific to SAYT requests.